src/test/java/com/practice/PRACTICE/service/EmpServiceImplTest.java

```diff
@@ -35,17 +35,17 @@ class EmpServiceImplTest {
     public void setup() {
       employee = Employee.builder()
               .empId(1)
-              .firstName("Sonam")
-              .lastName("Singh")
+              .firstName("abc")
+              .lastName("xyz")
               .build();
     }
     @DisplayName("JUnit Test for get All Employees")
     @Test
     public void getAllEmpTest() {
         Employee employee1 = Employee.builder()
                 .empId(2)
-                .firstName("Raj")
-                .lastName("Kumar")
+                .firstName("ahs")
+                .lastName("pqrs")
                 .build();
         given(empRepo.findAll()).willReturn(List.of(employee, employee1));
 
```

Comment:
Please add Unit test cases for negative scenarios as well, where exceptions can occur.